### PR TITLE
Mark 8.x as the Active LTS release

### PR DESCRIPTION
**Pull request by ntwb**
It's the 31st October 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,9 +8,9 @@
 |  v0.12.x  |**End-of-Life**|    -     |        -         |    2016-04-01     |   2016-12-31    |
 | [4.x][] |**Maintenance**| [Argon][]|    2015-10-01    |    2017-04-01     |   April 2018    |
 |  5.x    |No LTS         |          |                  |                   |                 |
-| [6.x][] |**Active**     | [Boron][]|    2016-10-18    |    April 2018     |   April 2019    |
+| [6.x][] |**Maintenance**     | [Boron][]|    2016-10-18    |    April 2018     |   April 2019    |
```
Comment (cjihrig) [inline README.md]: v6.x is still going to be Active LTS for 6 more months.

Comment (MylesBorins) [reply README.md]: + 1 to that!

Comment (ntwb) [reply README.md]: Fixed 👍

**Comment by ntwb** [comment]
Ha, there's a merge conflict and it looks like #272 snuck in whilst I whipped up this PR, as #272 comes before #273 and there's a proposal in #272 to change the verbiage further I'll close this PR in favour of that PR 😄

**Pull request closed without merging**